drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -2447,19 +2449,16 @@ Error RenderingDeviceVulkan::texture_update(RID p_texture, uint32_t p_layer, con
 }
 
 static _ALWAYS_INLINE_ void _copy_region(uint8_t const *__restrict p_src, uint8_t *__restrict p_dst, uint32_t p_src_x, uint32_t p_src_y, uint32_t p_src_w, uint32_t p_src_h, uint32_t p_src_full_w, uint32_t p_unit_size) {
-	uint32_t src_offset = (p_src_y * p_src_full_w + p_src_x) * p_unit_size;
-	uint32_t dst_offset = 0;
-	for (uint32_t y = p_src_h; y > 0; y--) {
-		uint8_t const *__restrict src = p_src + src_offset;
-		uint8_t *__restrict dst = p_dst + dst_offset;
-		for (uint32_t x = p_src_w * p_unit_size; x > 0; x--) {
-			*dst = *src;
-			src++;
-			dst++;
-		}
-		src_offset += p_src_full_w * p_unit_size;
-		dst_offset += p_src_w * p_unit_size;
-	}
+    uint32_t src_offset = (p_src_y * p_src_full_w + p_src_x) * p_unit_size;
```

Comment:
You need to fix your indentation, use tabs, not spaces, this and other style errors can be fixed by using [clang-format](https://docs.godotengine.org/en/stable/contributing/development/code_style_guidelines.html#using-clang-format-locally)